src/lib/logger.ts

```diff
@@ -5,24 +5,44 @@ import util from 'util'
 import chalk from 'chalk'
 import emojis from 'emojis'
 import wordWrap from 'word-wrap'
+import GleeMessage from './message'
+import GleeError from '../errors/glee-error.js'
 
 export { chalk }
 
+interface ILogLineWithIconOptions {
+  iconColor?: string,
+  textColor?: string,
+  highlightedWords?: string[],
+  disableEmojis?: boolean,
+  emptyLinesBefore?: number,
+  emptyLinesAfter?: number,
+}
+
+interface ILogOptions {
+  highlightedWords?: string[],
+}
+
+interface ILogErrorOptions {
+  highlightedWords?: string[],
+  showStack?: boolean,
+}
+
 const __filename = fileURLToPath(import.meta.url)
 const __dirname = dirname(__filename)
 const TSLogo = chalk.bgHex('#3178C6').white(' TS')
 
-const highlightWords = (words, text) => {
+const highlightWords = (words: string[], text: string) => {
   let result = text
 
   words.filter(Boolean).forEach(word => {
-    result = result.replace(new RegExp(word, 'g'), chalk.white(word))
+    result = result.replaceAll(word, chalk.white(word))
```

Comment:
This might be a problem, as it is [only supported in node > v15](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll).

Reply:
Fixed 👍 Good catch!